src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1606,7 +1595,7 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
             isVisible = helper.ReadBoolean("isVisible", true);
             isUpstreamVisible = helper.ReadBoolean("isUpstreamVisible", true);
             argumentLacing = helper.ReadEnum("lacing", LacingStrategy.Disabled);
-            IsSelectedInput = helper.ReadBoolean("isSelectedInput", true);
+            IsInputNode = helper.ReadBoolean("isSelectedInput", true);
```

Comment:
similar question as above, what happens here... if the serialized version was null from above.